### PR TITLE
Add red and yellow hair bow avatar accesorries

### PR DESCRIPTION
**Pull request by jakerockland**
##### Description

Adds red and yellow hair bows to avatar creator, re: #2333

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

avatar

##### Testing

Tested locally, screenshots attached:

screenshot 2018-10-20 02 17 59
screenshot 2018-10-20 02 18 39

screenshot 2018-10-20 02 18 05
screenshot 2018-10-20 02 18 26

##### Refers/Fixes

Refs: #2333

### CODE REVIEW

**Comment by jakerockland** [comment]
Sorry about the bit of messy commit history, squashed my commits :swea

File: app/assets/v2/css/fontawesome-all.min.css [hunk 1]
```diff
@@ -4,4 +4,4 @@
...
...format("embedded-opentype"),url(../webfonts/fa-brands-400.woff2) format("woff2"),url(../webfonts/fa-brands-400.woff) format("woff"),url(../webfonts/fa-brands-400.ttf) format("truetype"),url(../webfonts/fa-brands-400.svg#fontawesome) format("svg")}.fab{font-family:Font Awesome\ 5 Brands}@font-face{font-family:Font Awesome\ 5 Free;font-style:normal;font-weight:400;src:url(../webfonts/fa-regular-400.eot);src:url(../webfonts/fa-regular-400.eot?#iefix) format("embedded-opentype"),url(../webfonts/fa-regular-400.woff2) format("woff2"),url(../webfonts/fa-regular-400.woff) format("woff"),url(../webfonts/fa-regular-400.ttf) format("truetype"),url(../webfonts/fa-regular-400.svg#fontawesome) format("svg")}.far{font-weight:400}@font-face{font-family:Font Awesome\ 5 Free;font-style:normal;font-weight:900;src:url(../webfonts/fa-solid-900.eot);src:url(../webfonts/fa-solid-900.eot?#iefix) format("embedded-opentype"),url(../webfonts/fa-solid-900.woff2) format("woff2"),url(../webfonts/fa-solid-900.woff) format("woff"),url(../webfonts/fa-solid-900.ttf) format("truetype"),url(../webfonts/fa-solid-900.svg#fontawesome) format("svg")}.fa,.far,.fas{font-family:Font Awesome\ 5 Free}.fa,.fas{font-weight:900}
```
Comment (thelostone-mc) [inline app/assets/v2/css/fontawesome-all.min.css]: @jakerockland Any reason this file was changed?
If not can we reset it to what we have on master ?

**Comment by jakerockland** [comment]
Hrm, @thelostone-mc I updated my branch to fix re: the above convo, but for some reason the commits on this PR are not matching those on [my branch](https://github.com/gitcoinco/web/compare/master...jakerockland:hair-bow-accessories). Seems like a Github bug, have you seen this before?

**Comment by jakerockland** [comment]
@thelostone-mc Things are looking right now, Github was having some strangle issues last night 😅

**Pull request merged**